### PR TITLE
Fix EventLogLogger to include the logger name in the message

### PR DESCRIPTION
**Pull request by kichalla**
@davidfowl

### CODE REVIEW

**Comment by davidfowl** [comment]
Screen shot of how it looks? A beeline might be better

**Comment by kichalla** [comment]
Sure, examples:
image

image

New line? sure..but should it say like 

```
Logger: <logger-name-here>
<message-here>
```
or just
```
<logger-name-here>
<message-here>
```

**Comment by Eilon** [comment]
Either way is totally fine.

**Comment by Eilon** [comment]
That is, with a newline, but either format is fine.

**Comment by davidfowl** [comment]
```
<logger-name-here>
<message-here>
```

**Comment by kichalla** [comment]
updated. @Eilon @davidfowl

File: src/Microsoft.Framework.Logging.EventLog/EventLogLogger.cs [hunk 1]
```diff
@@ -94,6 +94,8 @@ public bool IsEnabled(LogLevel logLevel)
                 return;
             }
 
+            message = $"{_name}{Environment.NewLine}{message}";
```
Comment (Eilon) [inline src/Microsoft.Framework.Logging.EventLog/EventLogLogger.cs]: You know, you could just use `+` to concatenate :smile:

Comment (kichalla) [inline src/Microsoft.Framework.Logging.EventLog/EventLogLogger.cs]: yeah! :+1:

Comment (kichalla) [inline src/Microsoft.Framework.Logging.EventLog/EventLogLogger.cs]: updated @Eilon

**Comment by Eilon** [comment]
:shipit:

**Comment by kichalla** [comment]
commit d89a2abe6d4f0abe271f8b9d68ecc4dec4e4585d

**Pull request closed without merging**